translator.py

```diff
@@ -31,10 +31,14 @@ def render_pages():
         )
 
         locale = locale.split(".mo")[0]
+        lang_tag = locale.replace("_", "-")
 
         template = jinja_env.get_template("index.jinja2")
 
         translation = babel.support.Translations.load(localedir, [locale])
+        if locale != "en":
```

Comment:
Do we need this check even? I guess the fallback won't hurt with `en` either.

Reply:
Having the check won't hurt either :)